home-open-api/src/main/resources/yaml/openapi.yaml

```diff
@@ -1791,6 +1902,74 @@ components:
           maxLength: 13
           type: string
           example: "+380966617305"
+    CooperationDataScope:
+      type: string
+      enum:
+        - ALL
+        - ADRESS
+        - APARTMETS
+        - OWNERS
+    TaskStatus:
+      type: string
+      enum:
+        - PENDING
+        - ERROR
+        - PROCESSING
+        - ACCEPTED
+        - DECLINED
+    TaskType:
+      type: string
+      enum:
+        - IMPORT
+        - EXPORT
+    ReadTask:
```

Comment:
In Issue#292 property "data_scope" is absent. Maybe there should be perent model "ReadTask" which extends by ReadImportTask snd ReadExportTask?